index.html

```diff
@@ -193,7 +221,7 @@ <h2 class="title-footer">Contact me</h2>
           <input type="text" name="name" placeholder="Your Name" maxlength="30" required />
           <input type="email" name="email" placeholder="Your Email" required />
           <textarea name="message" rows="8" placeholder="Write your message here . . ." maxlength="500" required></textarea>
-          <button type="submit" class="btn">Get in touch</button>
+          <button type="submit" class="btn btn-1">Get in touch</button>
         </fieldset>
         </form>
         </footer>    
```

Comment:
- [ ] For the contact form, Please make sure all the elements of the form are centered on the page. I noticed your design is centered on some screen sizes and not on others. in order to clone the design we need to make sure all the content are centered on all the desktop screen sizes.

### Your design 
![image](https://github.com/jerymey/Portfolio-setup/assets/69077061/0e928014-4537-47ec-b7ed-5cea0669f674)